app.py

```diff
@@ -4,23 +4,31 @@
 import logging
 import uuid
 from dotenv import load_dotenv
-
+import httpx
 from quart import (
     Blueprint,
     Quart,
     jsonify,
     make_response,
     request,
     send_from_directory,
-    render_template
+    render_template,
```

Comment:
Just out of curiosity, are you using a particular formatter here for some of these formatting updates, or did you make these changes manually?

I have been thinking about standardizing on a code formatter as a best practice in the future, so mainly just wanted to ask about tooling you might be using here.

Reply:
i am using Black formatter in vscode Link: https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter